node/service/src/parachains_db.rs

```diff
@@ -19,19 +19,27 @@ use {
 	std::path::PathBuf,
 	std::sync::Arc,
 
-	kvdb::KeyValueDB,
+	kvdb::{DBTransaction, KeyValueDB},
+	parity_scale_codec::{Encode, Decode},
 };
 
 
 mod columns {
 	#[cfg(feature = "real-overseer")]
-	pub const NUM_COLUMNS: u32 = 3;
+	pub const NUM_COLUMNS: u32 = 4;
 
-	pub const COL_AVAILABILITY_DATA: u32 = 0;
-	pub const COL_AVAILABILITY_META: u32 = 1;
-	pub const COL_APPROVAL_DATA: u32 = 2;
+	// meta column for the database.
+	#[cfg(feature = "real-overseer")]
+	pub const COL_META: u32 = 0;
+
+	pub const COL_AVAILABILITY_DATA: u32 = 1;
+	pub const COL_AVAILABILITY_META: u32 = 2;
+	pub const COL_APPROVAL_DATA: u32 = 3;
 }
 
+const VERSION_KEY: &[u8] = b"version";
```

Comment:
I think the reason we're using a separate file for version instead of a key in the db is because in order to open the db you need to provide the correct amount of columns.

Reply:
I suggest we remove it for now, can be added later.

Reply:
Ahh, ok